pkg/dyninst/rcscrape/scraper.go

```diff
@@ -28,10 +30,18 @@ type Scraper struct {
 		sync.Mutex
 
 		sinks     map[ir.ProgramID]*scraperSink
+		processes map[actuator.ProcessID]*trackedProcess
 		debouncer debouncer
 	}
 }
 
+type trackedProcess struct {
+	pu procmon.ProcessUpdate
+	// The runtimeID reported by the tracer inside this process. Used for sanity
+	// checks.
```

Comment:
I don't understand "Used for sanity checks".

Reply:
removed; I had not understood the deal correctly